GPRI.ecl

```diff
@@ -0,0 +1,83 @@
+import python3 as python;
+IMPORT ML_Core.Types as MTypes;
+IMPORT Std.System.Thorlib;
+IMPORT $.Types as Types;
+IMPORT $.internal.rffGPR as rffGPR;
+
+// Common record structures
+layout_model2 := MTypes.layout_model2;
+nNodes := Thorlib.nodes();
+nodeId := Thorlib.node();
+initParams := Types.initParams;
+NumericField := MTypes.NumericField;
+
```

Comment:
Doesn't look like this bundle supports the Myriad interface.  We should note whether or not each bundle supports it, here or in README.